ui/app/components/toolbar.js

```diff
@@ -0,0 +1,6 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+  classNames: ['toolbar'],
+  tagName: 'nav',
+});
```

Comment:
So for all of these components, what would you think about them extending the `outer-html` component, and then moving all of the class stuff and tagName things to the templates? When we move to glimmer components, that will be the new way of doing things and I think it's nicer to just have to look at the template for HTML-y things and it's obvious if something is dynamic and then you'd look at the JS.

Reply:
I think that could work, and we could also just use something like `@type='add'` to get rid of the Add component too.